.github/workflows/deploy-dev.yml

```diff
@@ -1,4 +1,4 @@
-name: CI/CD pipeline - development backoffice
+name: Development backoffice
```

Comment:
Removed "CI/CD pipeline - " because it was truncating in some places on GitHub's UI as "CI/CD pipelin...", i.e. couldn't see _which_ pipeline.